lang/c++/impl/json/JsonDom.hh

```diff
@@ -35,14 +35,14 @@ class AVRO_DECL InputStream;
 
 namespace json {
 class Entity;
-    
+
```

Comment:
best to avoid whitespace changes, these are likely to be rejected by upstream

Reply:
Removed the white spaces.